components/dashboard/DashboardViews.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { omitBy } from 'lodash';
+
+import { Box } from '../Grid';
+import Tabs from '../StyledTabs';
+
+const omit = (query, keys) => {
+  return omitBy(query, (value, key) => !value || (keys && keys.includes(key)));
+};
```

Comment:
You already have this function in the parent component as `getQueryParams`. I suggest simplifying this component by just sending the filtered query down the line and removing the `omitMatchingParams` entirely.

Reply:
It is there because it's also used for each views query when seeing if there is a matching selected view - this is to enable changing the `orderBy` filter without popping out of the selected view - but that `orderBy` parameter still needs to be available in this component when selecting/applying a view.